### DISCUSSION TITLE
support jav sites

### DISCUSSION THREAD

**Discussion by sugizo** [EchterAlsFake/unofficial-api-for-pornhub#56] [Ideas]
**objective** 
support jav (japanese adult videos) sites 
like javscrapper python module on pypi

![image](https://github.com/EchterAlsFake/PHUB/assets/2418295/61dd61d5-aeff-43fa-b04b-7c0897823e24)

**Comment by EchterAlsFake** [comment]
Hi, thanks for your interest, but we won't do this. The PHUB library is only for PornHub. It's already very well optimized and the code is very strict. Supporting all the other sites would be soooooo much work we would need months for this and it would be nearly impossible to maintain. Maybe I write APIs for some of these sites for my Porn Fetch project but not for every site. 

Although you can use PHUB for massively downloading / scraping videos, we don't recommend this. Making an API for a website is legally a grey zone. Websites get more traffic than from a normal user and this leads to more server costs / more needed capacity. Please respect that when working on your project.
